charts/kubermatic-operator/templates/dockercfg.yaml

```diff
@@ -21,5 +20,4 @@ metadata:
     app.kubernetes.io/name: kubermatic-operator
 type: kubernetes.io/dockerconfigjson
 data:
-  .dockerconfigjson: {{ .Values.kubermaticOperator.imagePullSecret | b64enc | quote }}
-{{- end }}
+  .dockerconfigjson: {{ .Values.kubermaticOperator.imagePullSecret | default "{}" | b64enc | quote }}
```

Comment:
@xrstf  Wouldn't be cleaner to have the operator tolerating no `imagePullSecret` rather than creating an empty one as default?

Reply:
@irozzo-1A döne